solidity/contracts/deposit/Deposit.sol

```diff
@@ -425,10 +425,10 @@ contract Deposit is DepositFactoryAuthority {
         self.exitCourtesyCall();
     }
 
-    /// @notice     Notify the contract that the signers are undercollateralized.
+    /// @notice     Notify the contract that the signers are severely undercollateralized.
     /// @dev        Calls out to the system for oracle info.
-    function notifyUndercollateralizedLiquidation() public {
-        self.notifyUndercollateralizedLiquidation();
+    function notifySevereUndercollateralizedLiquidation() public {
+        self.notifySevereUndercollateralizedLiquidation();
```

Comment:
Actually quite tempted to take this opportunity to revamp all of our `notify*` naming, as it's not particularly consistent in how it describes what it's notifying… e.g. here, we're really doing `notifySeverlyUndercollateralized`, which *triggers* liquidation. Below, `notifyCourtesyTimeout` also triggers liquidation, but doesn't mention it in its name.

Let me see what timing overall we have, and then see if we merge as-is and follow up with broader renaming, or if we just do the broader renaming and close this PR. Thanks for calling attention here!